articles/what-is-a-custom-domain.mdx

```diff
@@ -0,0 +1,64 @@
+
+One, you cannot change the automatically generated testing domain name without creating a new Site on CloudCannon. The automatically generated domain may not be right for you or your Site. You can generate a new one by creating a new Site with the same files, however this can be inefficient.
+
+Two, by default CloudCannon automatically de-indexes Testing Domains so that they do not appear in search engines.
+
+## Using subdomains and subpaths
+
+You can separate and organize your website content into multiple Sites on CloudCannon and serve that content on a specific subdomain or subpath of your Custom Domain.
+
+* **Base Domain** — The unmodified domain name of a website, including the TLD (e.g., example.com, or cloudcannon.com). Also called a root or apex domain.
+* **Subdomain** — An additional name on the front of your URL, prefixing your base domain name (e.g., "docs" in docs.example.com, or "blog" in blog.example.com).
+* **Subpath** — An additional name prefixing your page slug and suffixing your base domain name (e.g., "articles" in example.com/articles/what-is-a-custom-domain/, or "staff" in example.com/staff/). Also called a subdirectory.
+
+Separating your content onto different CloudCannon Sites can be helpful to improve clarity for your content editors and users, and enable you to use separate Git repositories for distinct areas of your website.
+
+Let's walk through an example.
+
+We want to separate the content on our website for our team members and users to improve clarity around the layout of information on our website. To do this, we can create two CloudCannon Sites for our team members, and two subdomains for our users.
+
+* Site One: Contains our website's Marketing content, linked to our main domain (e.g., example.com). Linked to our Marketing Git repository, with branded page themes.
+* Site Two: Contains our website's Documentation content, linked to a subdomain (e.g., docs.example.com). Linked to our Documentation Git repository, with simpler page themes.
+
+We could also accomplish the same goal by hosting content on the dedicated subpaths (example.com and example.com/docs/). Whether you use subdomains or subpaths is a matter of personal preference.
+
+<comp.Notice info_type="info">
+  Having your content on separate Sites means you can specify which content different teams are allowed to edit using [Custom Permission Groups](/documentation/articles/what-are-custom-permission-groups/).
+</comp.Notice>
+
+When you have multiple Sites connected to subdomains, CloudCannon will redirect all unallocated subdomains to the main domain. For example, if you have one Site connected to example.com, and another Site connected to the subdomain docs.subdomain.com, CloudCannon will redirect anyone trying to access an unallocated subdomain (e.g., app.example.com) or misspelled subdomain name to the main domain, example.com.
```

Comment:
This is conditional of using CloudCannon DNS and having a DNS record pointing `*.example.com` to `sites.cloudcannon.com`.
The main domain itself can be configured on CloudCannon on Domains > Domain Settings > Redirects